packages/auth/src/OAuth.ts

```diff
@@ -24,7 +24,7 @@ class OAuth {
     args.authorization = args.authorization || CARTO_AUTHORIZATION_BASE;
     args.scopes = args.scopes || '';
 
-    this._emitter = new mitt();
+    this._emitter = new (mitt as any)();
```

Comment:
Doesn't it work with mitt()? What error does it throw?
Doing this is far from good 😁

Reply:
After applying the hoisting, the build was broken, throwing this:

```
@carto/toolkit-auth: Error: /Users/victor/CARTO/toolkit/packages/auth/src/OAuth.ts(27,21): semantic error TS7009: 'new' expression, whose target lacks a construct signature, implicitly has an 'any' type.
```

(using tsc 3.6.3)

Reply:
Oh, I remember the same thing using Airship. I don't know why we use here `new` keyword, it is not needed as [documentation](https://github.com/developit/mitt#typescript) shows:

```js
import mitt from 'mitt';
const emitter: mitt.Emitter = mitt();
```

Reply:
Done!